base_loadgen_program/code_axs.py

```diff
@@ -101,3 +101,10 @@ def generate_current_timestamp(used_for="unknown"):
     timestamp = datetime.datetime.now().strftime("%Y.%m.%dT%H:%M:%S")
     #print(f"GENERATING TIMESTAMP: {timestamp} used for: {used_for}")
     return timestamp
+
+def list_to_str(sdk_ver_list):
```

Comment:
@sahelib25, do you have any test cases for this code? I'd imagine that something more Python-ic like this would work:
```
def list_to_str(sdk_ver_list):
    sdk_ver_list_str = [ str(elem) for elem in sdk_ver_list ) ]
    return ".".join(sdk_ver_list_str)
```